### PR TITLE
Tickets/opsim 1093

### PR DESCRIPTION
**Pull request by yoachim**
Moving `scheduler`,`utils`, `site_utils`, and `skybrightness_pre` code over to a new repo.

### CODE REVIEW

File: rubin_sim/data/rs_download_data.py [hunk 1]
```diff
@@ -95,98 +81,11 @@ def rs_download_data():
...
-    for k in new_versions:
-        if len(k) <= 10:
-            sep = "\t\t"
-        else:
-            sep = "\t"
-        print(f"{k}{sep}{new_versions[k]}")
+    download_rubin_data(
+        files,
+        dirs=args.dirs,
+        print_versions_only=args.versions,
```
Comment (rhiannonlynne) [inline rubin_sim/data/rs_download_data.py:87]: This looks good, but since I just added the --upgrade flag (and it's really useful for workflows), would you mind adding it here as well? 
(I could always do this in a separate PR too).

File: rubin_sim/maf/plots/spatial_plotters.py [hunk 2]
```diff
@@ -177,6 +178,7 @@ def __call__(self, metric_value_in, slicer, user_plot_dict, fignum=None):
             )
             mask = np.zeros(metric_value.size)
             mask[np.where(metric_value == slicer.badval)] = 1
+            # XXX--hm, this looks broken
```
Comment (rhiannonlynne) [inline rubin_sim/maf/plots/spatial_plotters.py:181]: I'm not sure what you mean here - could you clarify?

File: tests/maf/test_batches.py [hunk 3]
```diff
@@ -4,13 +4,13 @@
 import unittest
 
 import numpy as np
+from rubin_scheduler.data import get_data_dir
+from rubin_scheduler.utils.code_utilities import sims_clean_up
```
Comment (rhiannonlynne) [inline tests/maf/test_batches.py:8]: What does sims_clean_up even do?

File: tests/maf/test_snmetrics.py [hunk 4]
```diff
@@ -151,7 +151,7 @@ def fake_data(band, season=1):
 class TestSNmetrics(unittest.TestCase):
     def setUp(self):
         if not os.path.isdir(os.path.join(get_data_dir(), "maf")):
-            self.skipTest("Skipping SN tests because running unit tests without full rubin_sim_data.")
+            self.skipTest("Skipping SN tests because running unit tests without full rubin_scheduler.data.")
```
Comment (rhiannonlynne) [inline tests/maf/test_snmetrics.py:154]: I think these may have been over-aggressive replacements?

**Review by rhiannonlynne — APPROVED** [review]
Looks good. 
I think the conda-forge version of Rubin-scheduler should be available shortly, so could wait and see how the unit tests go to be sure?

File: tests/maf/test_batches.py (hunk 3 above)
Comment (yoachim) [reply tests/maf/test_batches.py:8]: no idea

File: rubin_sim/maf/plots/spatial_plotters.py (hunk 2 above)
Comment (yoachim) [reply rubin_sim/maf/plots/spatial_plotters.py]: Below it was using something that wasn't imported. I probably fixed it, but this section still has no unit test coverage.
